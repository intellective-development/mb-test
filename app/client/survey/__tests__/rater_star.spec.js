import * as React from 'react';
import RaterStar from '../rater_star';

it('renders', () => {
  expect(render(
    <RaterStar />
  )).toMatchSnapshot();
});
