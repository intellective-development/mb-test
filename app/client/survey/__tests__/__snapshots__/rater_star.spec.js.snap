// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<span
  class="rating"
  role="button"
  tabindex="0"
>
  â˜…
</span>
`;
